### PR TITLE
ROPgadget: add page

### PR DESCRIPTION
**Pull request by johandegn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): v7.3**

### CODE REVIEW

File: pages/common/ropgadget.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`ROPgadget --binary {{path/to/binary}} --{{norop|nojob|nosys}}`
+
+- List gadgets in the binary not including bad bytes:
+
+`ROPgadget --binary {{path/to/binary}} --badbytes {{byte_string}}`
+
+- List gadgets in the binary with up to given bytes:
+
+`ROPgadget --binary {{path/to/binary}} --depth {{nbyte}}`
```
Comment (kbdharun) [inline pages/common/ropgadget.md:24]:
```suggestion
- List gadgets in the binary file:

`ROPgadget --binary {{path/to/binary}}`

- List gadgets in the binary file, filtered by regular expression:

`ROPgadget --binary {{path/to/binary}} --re {{regex}}`

- List gadgets in the binary file, excluding specified type:

`ROPgadget --binary {{path/to/binary}} --{{norop|nojob|nosys}}`

- List gadgets in the binary file, not including bad bytes:

`ROPgadget --binary {{path/to/binary}} --badbytes {{byte_string}}`

- List gadgets in the binary file, with up to a given number of bytes:

`ROPgadget --binary {{path/to/binary}} --depth {{nbyte}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Comment by johandegn** [comment]
I have made some small changes to match the ropper page better

**Review by mfrw — APPROVED** [review]
LGTM :)
Thanks @johandegn for your contribution.

**Pull request merged**